tesseract_ros/tesseract_rosutils/include/tesseract_rosutils/conversions.h

```diff
@@ -22,8 +22,6 @@ TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
 TESSERACT_COMMON_IGNORE_WARNINGS_POP
 
 #include <tesseract_motion_planners/core/waypoint.h>
-#include <tesseract_process_planners/process_definition.h>
```

Comment:
Waypoint has changed. Anything with tesseract_motion_planners namespace is probably broken.

Reply:
yep, I missed commenting out the utility function related to waypoints.

Reply:
This should be addressed now.